_dev/apps/ui/src/components/enhanced-conversions/enhanced-conversions-card.vue

```diff
@@ -57,40 +51,30 @@
 <script lang="ts">
 import {defineComponent} from 'vue';
 import {mapGetters} from 'vuex';
-import GettersTypesApp from '@/store/modules/app/getters-types';
 import GettersTypesCampaigns from '@/store/modules/campaigns/getters-types';
 import GettersTypesGoogleAds from '@/store/modules/google-ads/getters-types';
-import AlertEcReady from '@/components/enhanced-conversions/alert-ec-ready.vue';
 import AlertSignGadsTos from '@/components/enhanced-conversions/alert-sign-gads-tos.vue';
 
 export default defineComponent({
   name: 'EnhancedConversionsCard',
   components: {
-    AlertEcReady,
     AlertSignGadsTos,
   },
-  data() {
-    return {
-      showReadyToEnableFeature: false as boolean,
-    };
-  },
   computed: {
-    ...mapGetters('app', [
-      GettersTypesApp.GET_FEATURE_FLAG_ENHANCED_CONVERSIONS,
-    ]),
     ...mapGetters('campaigns', [
       GettersTypesCampaigns.GET_ENHANCED_CONVERSIONS_STATUS,
+      GettersTypesCampaigns.GET_ENHANCED_CONVERSIONS_TOGGABLE,
     ]),
```

Comment:
you can use mapGetters as an object to not reuse this computed to create another one =>

```suggestion
    ...mapGetters('campaigns', {
      GET_ENHANCED_CONVERSIONS_STATUS: GettersTypesCampaigns.GET_ENHANCED_CONVERSIONS_STATUS,
      enhancedConversionIsToggable: GettersTypesCampaigns.GET_ENHANCED_CONVERSIONS_TOGGABLE,
    }),
```
see documentation => https://vuex.vuejs.org/guide/getters.html#the-mapgetters-helper

Reply:
I removed the computed that was useless